apps/examples/testcase/le_tc/kernel/Kconfig

```diff
@@ -4,15 +4,15 @@
 #
 
 menuconfig EXAMPLES_TESTCASE_KERNEL
-	bool "Kernel TestCase Example"
+	bool "kernel TestCase Example"
```

Comment:
Could you let me know why you modify only `kernel`? There are many upper characters. I can't get your point for that.

Reply:
it was by mistake.